app/views/cloud_object_store_container/new.html.haml

```diff
@@ -0,0 +1,62 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "cloudObjectStoreContainerFormController"}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Basic Information')
+  .form-horizontal
+    .form-group{"ng-class" => "{'has-error': angularForm.storage_manager_id.$invalid}"}
+      %label.col-md-2.control-label
+        = _('Storage Manager')
+      .col-md-8
+        = select_tag("storage_manager_id",
+          options_for_select([["<#{_('Choose')}>", nil]] + @storage_manager_choices.sort, disabled: ["<#{_('Choose')}>", nil]),
```

Comment:
I need to get a list of those that support `cloud_object_store_container_create` so I've inserted this in the link above. And viola, yes, the list is correct! 👍 
```
/api/providers?expand=resources&filter[]=supports_cloud_object_store_container_create?=true

result:
{
   "name":"providers",
   "count":4,
   "subcount":1,
   "resources":[
      {
         "href":"http://172.16.117.115:3000/api/providers/4",
         "id":4,
         "name":"AWS S3 Storage Manager",
         "created_on":"2017-03-20T14:30:59Z",
         "updated_on":"2017-03-30T14:16:49Z",
         "zone_id":1,
         "type":"ManageIQ::Providers::Amazon::StorageManager::S3",
         "provider_region":"us-west-2",
         "last_refresh_date":"2017-03-30T14:16:49Z",
         "tenant_id":1,
         "parent_ems_id":1,
         "actions":[
            {
               "name":"edit",
               "method":"post",
               "href":"http://172.16.117.115:3000/api/providers/4"
            },
            {
               "name":"refresh",
               "method":"post",
               "href":"http://172.16.117.115:3000/api/providers/4"
            },
            {
               "name":"delete",
               "method":"post",
               "href":"http://172.16.117.115:3000/api/providers/4"
            },
            {
               "name":"delete",
               "method":"delete",
               "href":"http://172.16.117.115:3000/api/providers/4"
            }
         ]
      }
   ],
   "actions":[
      {
         "name":"query",
         "method":"post",
         "href":"http://172.16.117.115:3000/api/providers"
      },
      {
         "name":"create",
         "method":"post",
         "href":"http://172.16.117.115:3000/api/providers"
      },
      {
         "name":"edit",
         "method":"post",
         "href":"http://172.16.117.115:3000/api/providers"
      },
      {
         "name":"refresh",
         "method":"post",
         "href":"http://172.16.117.115:3000/api/providers"
      },
      {
         "name":"delete",
         "method":"post",
         "href":"http://172.16.117.115:3000/api/providers"
      }
   ]
}
```